### DISCUSSION TITLE
git config create => gets a 404 not found error

### DISCUSSION THREAD

**Discussion by RandallHoffpauir** [thoughtspot/cs_tools#87] [Q&A] [answered]
I am trying to setup the git config for cs_tools. When I do the "cs_tools tools git config create" command with the appropriate options I am getting a 404 not found error.  I am attaching the log file with some of the pertinent information changed.
Any help would be greatly appreciated!
[2023-12-05T08_10_07.log](https://github.com/thoughtspot/cs_tools/files/13562822/2023-12-05T08_10_07.log)

**Comment by RandallHoffpauir** [comment]
Has this been tested with Azure Git instead of GitHub? If I point it to a github repository it works fine, but if I point it to azure git repository I get the 404 error.

**Reply by boonhapus** [reply, marked as answer]
@RandallHoffpauir ThoughtSpot _currently_ only **[supports GitHub Personal and Enterprise](https://developers.thoughtspot.com/docs/git-integration)** for the VCS APIs. I'll check on timelines for other providers for you, I know we've talked about a few others internally as well.

**Reply by boonhapus** [reply]
I'll also make sure we're sanitizing those API parameters in the log file. 😅

**Reply by RandallHoffpauir** [reply]
Thanks, a timeline would be awesome.

**Reply by boonhapus** [reply]
@RandallHoffpauir - Can you upvote **[this Product Idea](https://community.thoughtspot.com/s/idea/0873n000000kGtiAAE/detail)** in the Community and also leave a comment mentioning Azure DevOps / TFS / Azure Repors or whichever flavor you're using?
